lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -73,6 +70,51 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * List the import conditions found in the prelude of an `@import` rule
+ *
+ * @param {import('postcss-value-parser').Node[]} params
+ * @returns {Array<string>}
+ */
+function listImportConditions(params) {
+	if (!params.length) return [];
+
+	/** @type {Array<Array<import('postcss-value-parser').Node>>} */
+	let media = [];
+	/** @type {Array<import('postcss-value-parser').Node>} */
+	let lastMediaQuery = [];
+
+	for (let i = 0; i < params.length; i++) {
```

Comment:
nit

you could put `params.length` in a const

Reply:
`.length` is only accessed twice and there is no performance penalty when accessing an array's length.

In this case I think it is fine to leave as is.
But happy to change if there is consensus that assigning to a const is better here :)

Reply:
`for...of` seems available to me 👀 

```js
for (const param of params) {
```

Reply:
I've changed this to `for...of`!

Thank you for the suggestion